docs/5g-on-helium/mining-5g.mdx

```diff
@@ -0,0 +1,162 @@
+---
+id: mining-5g
+hide_title: true
+sidebar_label: 5G Mining & Rewards
+sidebar_class_name: new
+slug: /5g-on-helium/mining-5g
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/mobile-header.png')} />
+
+# 5G Mining and MOBILE Rewards
+
+Several factors contribute to how much `MOBILE` a 5G Hotspot will earn over a given period, there
+are some higher level concepts, design elements, and rules to keep in mind that will help better
+explain what you might earn and why. This section covers these, as well as some mining and token
+reward basics.
+
+## How To Mine MOBILE
+
+Mining MOBILE and providing 5G coverage requires a Helium Hotspot and a compatible CBRS Small Cell
+Radio. These components can be purchased as a bundle or separately. 5G Hotspots must be online and
+registered in the SAS (Spectrum Access System) maintained by the FCC. Specific instructions can be
+found during onboarding from individual 5G Hotspot Makers.
+
+:::warning
+
+Misrepresentation of a CBRS radio’s installation information, including GPS location, height,
+antenna orientation or other characteristics, in a SAS is a violation of FCC regulations and may
+result in fines or criminal penalties.
```

Comment:
Are we sure about "criminal penalties"?

Reply:
change to and/or but yes, apparently: https://www.ecfr.gov/current/title-47/chapter-I/subchapter-D/part-95/subpart-A/section-95.313